config/global.ini.php

```diff
@@ -1095,6 +1095,7 @@
 type = BASIC ; proxy type for outbound/outgoing connections; currently, only BASIC is supported
 host = ; Proxy host: the host name of your proxy server (mandatory)
 port = ; Proxy port: the port that the proxy server listens to. There is no standard default, but 80, 1080, 3128, and 8080 are popular
+exclude = ; Hosts to exclude from proxy: optional. Comma separated list of hosts
```

Comment:
Guess we could mention that localhost is always excluded.
```suggestion
exclude = ; Comma separated list of hosts to exclude from proxy: optional; localhost is always excluded
```